src/lsp/lp_lsp.ml

```diff
@@ -382,36 +406,39 @@ let hover_symInfo ofmt ~id params =
   LIO.log_error "hoverSymInfo" sym_target;
   LIO.log_error "hoverSymInfo" (Range.interval_to_string interval); *)
 
-  (* The information about the tokens is stored *)
-  let sym = Pure.get_symbols doc.final in
-
-  (* The start/finish positions are used to hover the full qualified term,
-     not just the token *)
-  let start = Range.interval_start interval
-  and finish = Range.interval_end interval in
-
-  (* FIXME: types and typed conversion should take care of this *)
-  let sl, sc, fl, fc =
-    (Range.line start - 1),
-    (Range.column start - 1),
-    (Range.line finish - 1),
-    (Range.column finish - 1)
-  in
+  try
+    (* The information about the tokens is stored *)
+    let sym =
+      match doc.final with
+      | Some ss -> Pure.get_symbols ss
+      | None    -> raise (Error.fatal_no_pos("Horror")) in
```

Comment:
Just to know: will this error message be printed somewhere in the editor?

Reply:
No. The exception will be caught at the end of the function (line 456) and the message discarded. I changed the message though.